corehq/apps/locations/util.py

```diff
@@ -184,6 +185,7 @@ def _loc_type_dict(self, loc_type):
             uncategorized_keys.update(uncategorized_data.keys())
 
             loc_dict = {
+                'location_id': loc.location_id,
```

Comment:
I think it's fine - it only gets put in the export if it corresponds to a header in `tab_headers`.  It's small and cheap enough to compute that I don't think it'd be an issue.  (I tested it out both ways when I wrote this to be sure).